src/libraries/System.Runtime.Extensions/tests/System/BitConverterSpan.cs

```diff
@@ -46,131 +46,231 @@ public override void ConvertFromBool(bool boolean, byte[] expected)
 
         public override void ConvertFromShort(short num, byte[] expected)
         {
+            if (!BitConverter.IsLittleEndian)
+            {
+                expected = (byte[]) expected.Clone();
+                Array.Reverse(expected);
+            }
```

Comment:
Did you consider extracting this into a helper?

Reply:
OK, I'll do that.

Reply:
Patch updated.